mail_activity_team/tests/test_mail_activity_team.py

```diff
@@ -318,3 +320,21 @@ def test_schedule_activity_from_server_action(self):
         action.activity_team_id = self.team2
         action.with_context(active_model=partner._name, active_ids=partner.ids).run()
         self.assertEqual(partner.activity_ids[-1].team_id, self.team2)
+
+    def test_my_activity_date_deadline(self):
```

Comment:
Een korte docstring hierbij zou fijn zijn.

Reply:
Je mag dat anders als comment zetten op https://github.com/OCA/social/pull/<prevent crosslink to public PR>1280